.github/workflows/release.yaml

```diff
@@ -0,0 +1,41 @@
+name: Release
+on: 
+  push:
+    tags:
+      - 'v*'
+jobs:
+  image:
+    name: Push Container image
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-buildx-action@v1
+      - run: make docker-build
+      - name: Login to ghcr.io
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Push to ghcr.io
+        run: |
+          IMAGE_TAG=${GITHUB_REF#refs/tags/v} # Remove "v" prefix.
+          docker tag nyamber-controller:dev ghcr.io/cybozu-go/nyamber-controller:$IMAGE_TAG
+          docker push ghcr.io/cybozu-go/nyamber-controller:$IMAGE_TAG
+          docker tag localhost:5151/entrypoint:dev ghcr.io/cybozu-go/localhost:5151/entrypoint:$IMAGE_TAG
+          docker push ghcr.io/cybozu-go/localhost:5151/entrypoint:$IMAGE_TAG
```

Comment:
I don't think the image name ghcr.io/cybozu-go/localhost:5151/entrypoint is appropriate. For example, I think a name like ghcr.io/cybozu-go/nyamber-entrypoint would be better.

I am not sure if it is properly treated as a cybozu-go/nyamber package when pushed to ghcr with this image name.
We may need to set up the following link.
https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility#ensuring-workflow-access-to-your-package
https://docs.github.com/en/packages/learn-github-packages/connecting-a-repository-to-a-package#connecting-a-repository-to-a-container-image-using-the-command-line

Reply:
@kmdkuk 
> I am not sure if it is properly treated as a cybozu-go/nyamber package when pushed to ghcr with this image name.

As you say, you need to push the images to create a repository and set up the images to be updated from github 
actions.
I don't know whether these are done before or after PR merged.